src/kudu/consensus/leader_election.cc

```diff
@@ -156,53 +156,16 @@ void FlexibleVoteCounter::FetchTopologyInfo(
 
   CHECK(config_.has_commit_rule());
 
-  // A flag to denote if we could resolve a master.
-  // May involve cases of first election in the replicaset,
-  // last known master no longer part of the replicaset etc.
-  bool master_rule_unresolvable = false;
-
-  // Stores the requirement in the master region for a pessimistic
-  // computation of leader election quorum.
-  int master_region_req = -1;
+  // TODO(ritwikyadav) : This function will be simplified (or removed) shortly
```

Comment:
is this todo still relevant?

Reply:
Same comment as before. These TODOs get removed later in the stack. This is done this way to keep individual diff size low.